sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobTestBase.java

```diff
@@ -257,6 +257,12 @@ protected String setupBlobMatchCondition(BlobClientBase bc, String match) {
         }
     }
 
+    protected static List<String> convertNulls(String lease, String match) {
```

Comment:
I'm wondering if we could update the signature here to accept [varargs](https://docs.oracle.com/javase/8/docs/technotes/guides/language/varargs.html) to accept any number of string conditions provided to it?

I think that could be beneficial because if we ever introduce a third condition string and want to parameterize against it, we would not need to update all of the tests to make it compatible.